README.md

```diff
@@ -117,4 +117,198 @@ containers:
                name: test-atlas-operator-project-test-cluster-theuser
                key: connectionStringStandardSrv
 
-```
\ No newline at end of file
+```
+### Step 4. Test Database as a Service (DBaaS) on OpenShift
+
+The Atlas Operator is integrated with the [Red Hat Database-as-a-Service (DBaaS) Operator](https://github.com/RHEcosystemAppEng/dbaas-operator) which allows application developers to import database instances and connect to the databases through the [Service Binding Operator](https://github.com/redhat-developer/service-binding-operator). More information can be found [here](https://github.com/RHEcosystemAppEng/dbaas-operator#readme).
+
+Note that both the DBaaS Operator and Atlas Operator should be installed through the [Operator Lifecyle Manager (OLM)](https://github.com/operator-framework/operator-lifecycle-manager).
+
+**1.** Check DBaaS Registration
+
+If the DBaaS Operator has been deployed in the OpenShift Cluster, the Atlas Operator automatically creates a cluster level [DBaaSProvider](https://github.com/RHEcosystemAppEng/dbaas-operator/blob/main/config/crd/bases/dbaas.redhat.com_dbaasproviders.yaml) customer resource (CR) object `mongodb-atlas-registration` to automatically register itself with the DBaaS Operator. 
```

Comment:
s/customer/custom missed this one earlier, sorry :)

Reply:
Corrected